api/internal/user/entity/student.go

```diff
@@ -23,6 +34,43 @@ type Student struct {
 
 type Students []*Student
 
+func (s *Student) Fill(now time.Time) {
+	year := int64(now.Year())
+	month := int(now.Month())
+	age := year - s.BirthYear
+	if month < 4 {
```

Comment:
`age = {今の年度} - s.BirthYear` ってすれば、このあとのif文いらなくなる気がした

今の年度を計算するためのMethodを実装してみたので、mainブランチの状態を反映してから↓のに書き換えるともう少しすっきりできそう...?

```suggestion
	year := jst.FiscalYear(now)
	age := year - s.BirthYear
```